src/database/auth.rs

```diff
@@ -224,24 +253,20 @@ impl super::DatabaseWrapper {
 
         let user_id_arg_clone = user_id_arg.clone();
```

Comment:
Same thing here, is this clone necessary?

Reply:
It's not, thanks